Comparison Engine/Comparison Engine/Form1.cs

```diff
@@ -119,27 +132,34 @@ private void searchFunction(String keyword)
         //OPENING CHILD FORMS    #childforms
 
         public Form activeForm = null;
+        public MapForm mainMapForm = null;
```

Comment:
Shouldn't we hold all the class variables at the top?

Reply:
Done